_drafts/SC2-4-4-link-has-name.md

```diff
@@ -35,51 +35,61 @@ Each target element has an [accessible name][] that is [non-empty][].
 
 There are no major accessibility support issues known for this rule.
 
-@@@TO DO: Accessibility Support in SVG???
-
 ## Background
 
-- Links to Techniques for WCAG 2.0
-- Latest version: Techniques for WCAG 2.0 W3C Working Group Note 8 April 2014
-- Link to other methodologies, test specifications or tools
-- Links to test cases, test suites, unit tests, etc.
+- [https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=244#navigation-mechanisms-refs](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=244#navigation-mechanisms-refs)
+- [ARIA7: Using aria-labelledby for link purpose](https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html)
+- [ARIA8: Using aria-label for link purpose](https://www.w3.org/TR/WCAG20-TECHS/ARIA8.html)
+- [F89: Failure of Success Criteria 2.4.4, 2.4.9 and 4.1.2 due to using null alt on an image where the image is the only content in a link](http://www.w3.org/TR/WCAG20-TECHS/F89.html)
 - The WCAG 2.0 Techniques already contain examples and code snippets to illustrate which content passes or fails the test. Whenever possible auto-wcag refers to those. Another source for test cases is the W3C Before and After Demonstration.
-- Other references
 
 ## Test Cases
 
 ### Passed
 ```html
-<a href="http://www.w3.org/WAI">Web Accessibility Initiative (WAI)</a>
+<a href="http://www.w3.org/WAI"> Web Accessibility Initiative (WAI) </a>
 ```
 
 ```html
 <button role="link">Click me!</button>
 ```
 
```

Comment:
I suggest adding passing test cases for at least these cases: 
- Link containing only an image (which I am pretty sure will be applicable to this rule, as it is written right now)
- Link containing both image and text (which I am pretty sure will be applicable to this rule, as it is written right now)
- Link labelled via e.g. aria-label, aria-labelledby, title etc.